sections/SOTA.tex

```diff
@@ -44,18 +43,17 @@ \subsection{Julia}
 Julia is a high-level, high-performance dynamic programming language for technical computing, with syntax familiar to users of other technical computing environments.
 Julia is designed for parallelism and cloud computing; making it efficient and easy to use for these tasks.
 Julia has a high level of abstraction because the user only needs a single keyword (\@parallel) for it to do the calculation in parallel.
-The code is therefore looking clean without any boilerplate and there is a high level of readability.
+The code is therefore looking clean without any boilerplate and there is a high level of readability\todo{Vi har ikke skrevet om readability de andre steder. Er det ligemeget?}.
 Julia uses both OpenCL and CUDA as backend making it very compatible and easy to use on different systems and devices.\citep{Julia_Git,Julia}
 
 \input{figures/MathGPULanguages.tex} 
 
 \subsection{Conclusion}  
 
-The different languages mentioned in this section all have some way of communicating with the GPU using both CUDA and OpenCL, and call also be seen on \myref{tbl:sota} .
-They all require the user to specify when to use the GPU instead of the CPU, and the level of abstraction varies, where some require a lot of control of the programmer, and others require specific function calls to GPU functions. 
+The different languages mentioned in this section all have some way of communicating with the GPU using both CUDA and OpenCL, which also can be seen on \myref{tbl:sota} .
+They all require the user to specify when to use the GPU instead of the CPU, and the level of abstraction varies, where some require a lot of control of the programmer, and others require specific function calls to GPU functions\todo{Sidste del her er lidt underligt}.
 They are all usable for general purpose programming, except the libraries for R, which only provides the programmer with specific R functions to be computed on the GPU.
-For scientific computing, especially oriented towards matrices and vector calculations, these are all very viable options.
-The languages do require some knowledge of them, and if one were to choose a language with low level of abstraction, one would need knowledge of allocating memory etc.
+For scientific computing, especially oriented towards linear algebra with matrices and vector calculations, these are all very viable options.
```

Comment:
Måske bedre at vende den indskutte sætning om, således sætter vi mere fokus på matrice og vektor beregning frem for et helt felt. " [...], especially oriented towards matrice and vector calculations such as linear algebra includes, [...]"

Fjern very.